examples/apache-directory-list-local/webdav.conf

```diff
@@ -13,4 +13,15 @@
         Options +Indexes
         HeaderName /webdav-js/examples/apache-directory-list-local/header.html
     </Location>
+
+    # Add security
+    <Directory /var/www/webdav-js>
+        Options +Indexes
```

Comment:
I don't think we need this for the `webdav-js` folder.

Reply:
https://github.com/dom111/webdav-js/pull/21#discussion_r233023179